### PR TITLE
Refactor database settings 

### PR DESCRIPTION
**Pull request by wmemorgan**
Replace PostgreSQL database settings with environment variables

### CODE REVIEW

**Comment by wmemorgan** [comment]
[Install backend boilerplate on all local workstations](https://trello.com/c/F23PD3nX/26-install-backend-boilerplate-on-all-local-workstations)

File: .gitignore [hunk 1]
```diff
@@ -12,4 +12,5 @@ __pycache__/
 local_settings.py
 db.sqlite3
 
-tmp
\ No newline at end of file
+tmp
+Pipfile.lock
```
Comment (LDinho) [inline .gitignore]: @wmemorgan don't think we want to add the Pipfile.lock into .gitignore.
Also, we may not want to completely delete the Pipfile.lock?

**Review by LDinho — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (wmemorgan) [reply .gitignore]: I didn't see a Pipfile.lock in the original repo so I thought we should keep it out. But I did some Googl-ing and it seems that is generally recommended to keep Pipfile.lock in version control despite having a different OS among team members. Here's the thread I found that discusses it:

https://github.com/pypa/pipenv/issues/598

Comment (LDinho) [reply .gitignore]: I see. Sounds good!

**Review by LDinho — APPROVED** [review]

**Pull request merged**